design-collaboration-guidelines.md

```diff
@@ -0,0 +1,82 @@
+# Design Collaboration Guidelines
+
+As an app developer to provide the best version of the design, you need to define a standard vocabulary between the design team and you.
```

Comment:
I'd rewrite this as follows:

```
As an app developer, providing the best implementation of the design requested must be one of your priorities. To accomplish this goal, you need to define a standard vocabulary between the design team and you.
```

What do you think?

Reply:
Yes, I will just change the person, as I want this be written as Karumi perspective.